samples/event_manager_proxy/README.rst

```diff
@@ -174,7 +174,7 @@ After programming the sample to your development kit for both cores, test it by
         [00:00:07.015,502] <inf> event_manager: e: ack_event
 
 Now all the measurement and control events are generated on the remote core and passed to the host core, where the statistics are generated.
-On the remote core, we can also see an :c:type:`ack_event`, which is not passed to the host core.
+On the remote core, you can also see an :c:type:`ack_event` that is not passed to the host core.
```

Comment:
```suggestion
On the remote core, you can also see an event :c:type:`ack_event` that is not passed to the host core.
```

Reply:
Might be a bit redundant to say event before ack_event.